rabbitmq/src/urabbitmq/impl/io/isocket.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <cstddef>
+
+#include <userver/engine/io/socket.hpp>
+#include <userver/engine/io/tls_wrapper.hpp>
+
+USERVER_NAMESPACE_BEGIN
+
+namespace urabbitmq::impl::io {
+
+class ISocket {
```

Comment:
Now we have RwBase class that does the same thing

Reply:
Good news, def. will use that